debugging.md

```diff
@@ -12,6 +12,9 @@ applications.
 Before running your applications, test your code with `flutter
 analyze`. This tool (which is a wrapper around the `dartanalyzer`
 tool) will analyze your code and help you find possible mistakes.
+If you're using the
+[https://atom.io/packages/flutter](Flutter plugin for Atom), this
```

Comment:
Thanks @a14n.  Fixed in https://github.com/flutter/flutter.github.io/commit/199f6d640c5f880fc2a06db853cd00e069c6a4f0